roboteam_ai/src/conditions/IsRobotClosestToBall.cpp

```diff
@@ -16,16 +17,20 @@ IsRobotClosestToBall::IsRobotClosestToBall(std::string name, bt::Blackboard::Ptr
 : Condition(std::move(name), std::move(blackboard)) { }
 
 bt::Node::Status IsRobotClosestToBall::onUpdate() {
-    Vector2 ballPos(ball->pos);
-
-    if (properties->hasDouble("secondsAhead")) {
+    Vector2 ballPos;
+    if (properties->getBool("atBallStillPosition")) {
+        ballPos = ball->getBallStillPosition();
```

Comment:
I am not a fan of blackboards and I think a seperate skill would be easier to read the trees later on